TCAT/Controllers/ActionOnboardViewController.swift

```diff
@@ -59,6 +59,37 @@ class ActionOnboardViewController: UIViewController, CLLocationManagerDelegate {
         view.addSubview(description)
         view.addSubview(image)
         view.addSubview(secondButton)
+        
+        image.backgroundColor = .clear
+        image.snp.makeConstraints { (make) in
+            make.width.equalTo(311)
+            make.height.equalTo(325.5)
+            make.centerX.equalToSuperview()
+            make.top.equalToSuperview().offset(32)
+        }
+        image.image = getImage()
+        
+        title.font = UIFont(name: FontNames.SanFrancisco.Bold, size: 28)
+        title.textColor = UIColor.primaryTextColor
+        title.text = getTitle()
+        title.center = view.center
+        title.textAlignment = .center
+        title.snp.makeConstraints { (make) in
+            make.bottom.equalToSuperview().offset(-244)
```

Comment:
These values are on the larger side @AAAstorga

Reply:
Imma have to fix this–I think I didn't merge properly with master, this is def not right.